openweather-based-bidding/main.gs

```diff
@@ -45,6 +48,19 @@ function monitorWeatherAndSyncWithDV360() {
     const row = rows[i];
     const iPlus1 = i + 1;
 
+    // Check if we already processed this item
+    const currentDateTime = new Date();
+    const lastUpdated = new Date(
+      row[ config.getHeaderIndex('col-last-updated') ]
+    );
+    
+    const diff = (currentDateTime - lastUpdated) / 1000 / 60 / 60;
+    const howOften = parseInt(config.get('how-often-to-check-in-hours'));
+    if ( !onlyCheckAPI && howOften && diff < howOften) {
```

Comment:
Let's be consistent:
`if ( !onlyCheckAPI...` vs.
`if (! onlyCheckAPI...` vs.
`if (!sheetsApi.write...`

I'd prefer the last one.

Also, according to my comment above:
`hoursBetweenUpdates`

Reply:
Totally agree, will review all if statements. Also would be nice to agree on the coding guidelines.